students/TrkCat/Lesson07/html_render.py

```diff
@@ -0,0 +1,116 @@
+class Element:
+    tag = 'html'
+    indent = 4
+
+    def __init__(self, content=None, **kwargs):
+        if content is None:
+            self.content = []
+        else:
+            self.content = [content]
+        self.attr = kwargs
+
+    def append(self, string):
+        self.content.append(string)
+
+    def render(self, file_out, cur_ind=None):
+        self.render_open_tag(file_out, cur_ind, self.attr)
+        file_out.write('\n')
+        ind = 0 if cur_ind is None else cur_ind
+        for item in self.content:
+            if hasattr(item, 'render'):
+                item.render(file_out, ind + 1)
+            else:
+                file_out.write(self.conv_ind(ind + 1) + item + '\n')
+        file_out.write(self.conv_ind(cur_ind))
+        self.render_close_tag(file_out)
+
+    def render_open_tag(self, file_out, cur_ind=None, attr={}):
```

Comment:
Be careful with mutable default arguments, better to set `attr` to `None` and handle with the same pattern you use for `content` argument on `__init__`